content/reliability/docs/dataplane.md

```diff
@@ -215,7 +215,7 @@ CoreDNS fulfills name resolution and service discovery functions in Kubernetes.
 
 ## Recommendations
 ### Monitor CoreDNS metrics
-CoreDNS has built in support for [Prometheus](https://github.com/coredns/coredns/tree/master/plugin/metrics). You should especially consider monitoring CoreDNS latency (`coredns_dns_request_duration_seconds_sum`), errors (`coredns_dns_response_rcode_count_total`, NXDOMAIN, SERVFAIL, FormErr) and CoreDNS Pod’s memory consumption.
+CoreDNS has built in support for [Prometheus](https://github.com/coredns/coredns/tree/master/plugin/metrics). You should especially consider monitoring CoreDNS latency (`coredns_dns_request_duration_seconds_sum`), errors (`coredns_dns_responses_total`, NXDOMAIN, SERVFAIL, FormErr) and CoreDNS Pod’s memory consumption.
 
```

Comment:
Can you add a comment that before 1.7.1 the metric was called `core_dns_response_rcode_count_total`? We can't assume that everyone will be running this release. Please add a link to https://github.com/coredns/coredns/blob/master/notes/coredns-1.7.1.md too so people can find where this change is documented.

Reply:
Sure, I just added a comment. Please review it again